jobs/build/release-branch/Jenkinsfile

```diff
@@ -0,0 +1,108 @@
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+  if(env.WORKSPACE == null) {
+    env.WORKSPACE = pwd()
+  }
+}
+
+def ssh(cmd) {
+  sh "ssh openshiftdevel -- bash -x <<'EOF'\n${cmd}\nEOF"
+}
+
+def mail_failure(err, approvers) {
+  mail(
+    to: approvers, from: "aos-cd@redhat.com",
```

Comment:
Unfortunately, there doesn't seem to be a way to do that in Jenkins.

But the approvers are going to be a very small (1?) set of people, right?